### PR TITLE
Fix functional tests in aat staging - missing env var

### PR DESCRIPTION
**Pull request by doncem**
### Change description ###

PR and aat staging are using different queues. Env vars already defined just missing this config

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/application.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ azure:
       namespace: ${PAYMENTS_QUEUE_NAMESPACE}
       write-access-key: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY}
       write-access-key-name: ${PAYMENTS_QUEUE_WRITE_ACCESS_KEY_NAME}
-      queue-name: payments
+      queue-name: ${PAYMENTS_QUEUE_NAME}
```
Comment (nitinprabhu) [inline src/functionalTest/resources/application.yaml:15]: you need to set value for this env var in jenkins cnp ?

Comment (doncem) [reply src/functionalTest/resources/application.yaml:15]: true. it's present for app only :facepalm:

Comment (nitinprabhu) [reply src/functionalTest/resources/application.yaml:15]: Also you will need to set env var in flux for functional tests. Worth adding that first and then merging this so both staging and non staging func tests works

Comment (doncem) [reply src/functionalTest/resources/application.yaml:15]: will have to include in existing chart bump

**Pull request merged**